cli/Makefile

```diff
@@ -0,0 +1,16 @@
+build:
+		go build -o bin/tobs .
+
+test:
+		go test ./tests/
+
+go-vet:
```

Comment:
can we add these checks to the github workflow configs?

Reply:
I will add them to `github actions configs` too, having these checks in Makefile would make local testing much easier before pushing the code.